pkg/virt-launcher/virtwrap/util/devices_utils.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+// GetVolumeNameByTarget returns the volume name associated to the device target in the domain (e.g vda)
+func GetVolumeNameByTarget(domain *api.Domain, target string) string {
```

Comment:
we have so many of these util files... 
I wonder do we need to create a new file or we can just keep this function in the convertor since this is not being used anywhere else right now?

Reply:
Yes, I was also not very satisfied. I can try to move it :)